week2-section1/exercise4/Main.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+* Create a printer class. This class should allow you to store a string and then output it in different formats. Ask the user to input a string and then show what your printer can do! Remember to use char arrays instead of strings and avoid the std namespace.
+*/
+
+#include <iostream>
+#include<string>
+
+class Printer
+{
+private:
+	std::string WordEntry;
+public:
+	 Printer()
+	{
+		WordEntry = " ";
+	};
+	Printer(std::string OutputString) {
```

Comment:
Same comment about braces. Please update the whole review to follow this standard.